src/app/components/champion/ChampionView.tsx

```diff
@@ -70,31 +60,13 @@ export const ChampionView: FC<ChampionProps> = ({champion, callback, championsLi
         }
     }))(Tooltip)
 
-    function cleanDescription(description:string) {
+    function cleanDescription(description: string) {
```

Comment:
in utils.ts ? Can you give me an example of the change that your regex do in the comment ? It would be nice.

Reply:
Added a comment explaining the effect of the regex and moved it to a more fitting location.